build.sbt

```diff
@@ -0,0 +1,41 @@
+import Dependencies._
+
+ThisBuild / scalaVersion     := "2.13.2"
```

Comment:
Scala 2.13.3 is available. Might as well start out the project on the latest version.

Reply:
Ah cool. Everything still worked after I changed it which was surprising :smile: